src/components/App.jsx

```diff
@@ -0,0 +1,40 @@
+//Components
+import Head from './Head';
+import ListButtons from './ListButtons';
+import ListItems from './ListItems';
+import NewButton from './NewButton';
+
+//React Hooks
+import { useState } from "react";
+
+//Test database
+import db from "./testDB";
+
+const App = () => {
+    const [database, setDatabase] = useState(db);
+
+
+    const fetchGroceryList = async () => {
+        const resp = await fetch("http://localhost:4001/", {
```

Comment:
Note: This is just example code. The final URL will be different. I won't review this as part of this PR...